dist/connection-manager.js

```diff
@@ -0,0 +1,231 @@
+"use strict";
+/**
+ * Copyright (C) Macrometa, Inc - All Rights Reserved
+ *
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ * Written by Macrometa, Inc <product@macrometa.com>, May 2024
+ */
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.ConnectionManager = void 0;
+const types_1 = require("./types");
+const query_set_1 = require("./query-set");
+const switchable_connection_1 = require("./switchable-connection");
+const filters_state_1 = require("./filters-state");
+/**
+ * The main class manages connection and queries.
```

Comment:
Maybe it is worth only having the documentation in the interface to not duplicate comments for the same methods

```suggestion
 * The main class manages the connection and queries.
```

Reply:
This file is not using at all, deleted